Simplified/LibraryFinder/NYPLLibraryFinderBusinessLogic.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  NYPLLibraryFinderBusinessLogic.swift
+//  Simplified
+//
+//  Created by Ernest Fan on 2021-04-21.
+//  Copyright © 2021 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+
+protocol NYPLLibraryFinderDataProviding {
+  var userAccounts: [Account] { get }
+  var newLibraryAccounts: [Account] { get }
+  
+  // Might need to add pagination to the request
+  func requestLibraryList(searchKeyword: String?, completion: @escaping (Error?) -> ())
+}
+
+class NYPLLibraryFinderBusinessLogic: NSObject, NYPLLibraryFinderDataProviding {
+  var userAccounts: [Account]
+  var newLibraryAccounts: [Account]
```

Comment:
Are userAccounts and newLibraryAccounts going at change with the state of the class? If not, can we have them as constants instead?

Reply:
`newLibraryAccounts` will be updated when user request the data from server, `userAccounts` might not, I will leave a comment and take a second look later when other parts of Finder work are completed